src/openzaak/components/zaken/tests/test_notifications_kanaal.py

```diff
@@ -23,60 +25,119 @@ def setUpTestData(cls):
         site.domain = "example.com"
         site.save()
 
-    @patch(
-        "vng_api_common.notifications.management.commands.register_kanaal.get_client"
-    )
+        cls._configure_notifications()
+
+        kanaal = Kanaal(label="dummy-kanaal", main_resource=Zaak)
+        cls.addClassCleanup(lambda: KANAAL_REGISTRY.remove(kanaal))
+
+    @patch("notifications_api_common.models.NotificationsConfig.get_client")
     def test_kanaal_create_with_name(self, mock_get_client):
         """
         Test is request to create kanaal is send with specified kanaal name
         """
         client = mock_get_client.return_value
         client.list.return_value = []
-        # ensure this is added to the registry
-        Kanaal(label="kanaal_test", main_resource=Zaak)
 
         stdout = StringIO()
         call_command(
-            "register_kanaal",
-            "kanaal_test",
-            notificaties_api_root="https://example.com/api/v1",
-            stdout=stdout,
+            "register_kanalen", kanalen=["dummy-kanaal"], stdout=stdout,
         )
 
         client.create.assert_called_once_with(
             "kanaal",
             {
-                "naam": "kanaal_test",
-                "documentatieLink": "https://example.com/ref/kanalen/#kanaal_test",
+                "naam": "dummy-kanaal",
+                "documentatieLink": "https://example.com/ref/kanalen/#dummy-kanaal",
                 "filters": [],
             },
         )
 
-    @patch(
-        "vng_api_common.notifications.management.commands.register_kanaal.get_client"
-    )
-    @override_settings(NOTIFICATIONS_KANAAL="dummy-kanaal")
+    @patch("notifications_api_common.models.NotificationsConfig.get_client")
     def test_kanaal_create_without_name(self, mock_get_client):
         """
-        Test request to create kanaal is sent with default kanaal name
+        Test is request to create kanaal is send for all registered kanalen
         """
         client = mock_get_client.return_value
         client.list.return_value = []
-        # ensure this is added to the registry
-        Kanaal(label="dummy-kanaal", main_resource=Zaak)
 
         stdout = StringIO()
         call_command(
-            "register_kanaal",
-            notificaties_api_root="https://example.com/api/v1",
-            stdout=stdout,
+            "register_kanalen", stdout=stdout,
         )
 
-        client.create.assert_called_once_with(
-            "kanaal",
-            {
-                "naam": "dummy-kanaal",
-                "documentatieLink": "https://example.com/ref/kanalen/#dummy-kanaal",
-                "filters": [],
-            },
+        client.create.assert_has_calls(
```

Comment:
This test now seems to be duplicated a couple of times - imo it should be moved just to the non-component location (`openzaak.notifications` seems appropriate) and removed in the other places.